packages/authorize/src/Authorize.stories.tsx

```diff
@@ -36,47 +36,51 @@ export default {
     authorized: 'You are authorized to see this content.',
     region: true,
   },
-} as Meta;
-
-export const Default: Story = ({ authorized, loader, negate, organizationId, permissions, region, unauthorized }) => (
-  <div>
-    <p>
-      For this demo, the following permissions are granted: 1234, 2345, 3456, 4567, 5678, 6789. You can use the knobs to
-      see what the component will do when you set the required permissions to various things.
-    </p>
-    <hr />
-    <Authorize
-      permissions={permissions}
-      organizationId={organizationId}
-      negate={negate}
-      loader={loader}
-      unauthorized={<Alert color="danger">{unauthorized}</Alert>}
-      region={region}
-    >
-      <Alert color="success">{authorized}</Alert>
-    </Authorize>
-  </div>
-);
-Default.args = {
-  negate: false,
-  loader: true,
+  component: Authorize,
 };
-Default.storyName = 'default';
-
-export const UseAuthorize: Story = ({ authorized, organizationId, permissions, region, unauthorized }) => {
-  const { authorized: isAuthorized, isLoading } = useAuthorize(permissions, { organizationId, region });
 
-  return (
+export const _Authorize: StoryObj<typeof Authorize> = {
+  render: ({ authorized, loader, negate, organizationId, permissions, region, unauthorized }) => (
     <div>
       <p>
         For this demo, the following permissions are granted: 1234, 2345, 3456, 4567, 5678, 6789. You can use the knobs
         to see what the component will do when you set the required permissions to various things.
       </p>
       <hr />
-      <BlockUi blocking={isLoading} renderChildren={false}>
-        {isAuthorized ? <Alert color="success">{authorized}</Alert> : <Alert color="danger">{unauthorized}</Alert>}
-      </BlockUi>
+      <Authorize
+        permissions={permissions}
+        organizationId={organizationId}
+        negate={negate}
+        loader={loader}
+        unauthorized={<Alert color="danger">{unauthorized}</Alert>}
+        region={region}
+      >
+        <Alert color="success">{authorized}</Alert>
+      </Authorize>
     </div>
-  );
+  ),
+  args: {
+    negate: false,
+    loader: true,
+  },
+};
+
+export const _UseAuthorize: StoryObj<typeof BlockUi> = {
```

Comment:
> The _UseAuthorize StoryObj has a typeof BlockUI, but the name of the story should be 'useAuthorize' right? That is why I left the storyName in it, but I wasn't sure if that was the correct thing.

afaik the type isn't super important if you're not relying on storybook to infer the args/controls, but since you have them explicitly listed I would assume it's alright. Does anything odd jump out when you view the story through the canvas (go to the canvas tab for the story to see the associated controls with the story)

Reply:
Nothing odd jumps out to me.